tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "strictPropertyInitialization": false,
     "types": ["webpack-env", "jest"],
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": ["src/*"],
+      "@lib/*":["src/lib/components/*"]
```

Comment:
It looks like you haven't used @lib anywhere in the code. Why is it added here?

Reply:
this is created for future use from now we could use @lib directly every where whenever we are using anything from lib/components

Reply:
I understand that. You have '@/lib/components' used in some places. You can replace them with '@lib'. Also, I would suggest you use some other short name instead of '@lib' coz the path itself contains '/lib'. You can use something like '@sgt' or '@sgtsfc' or something similar to that.

Reply:
I previously used sgt keyword but was said by ajay to avoide as of now till we decide final